### DISCUSSION TITLE
Windows shows offline but internet is working fine

### DISCUSSION THREAD

**Discussion by claudio-klez** [DNSCrypt/dnscrypt-proxy#1721] [Q&A]
Since I enabled SimpleDNSCrypt as a service, Windows NLA Service has been broken, showing the PC as always offline. I can't seem to make it work, maybe because I'm using the wrong parameters in the registry. I think DNSCrypt should have an option to fix NLA Service by properly modifying the parameters mentioned in the reference page below:
[REFERENCE](
https://answers.microsoft.com/en-us/windows/forum/windows_10-networking/windows-shows-no-internet-access-but-my-internet/2e9b593f-c31c-4448-b5d9-6e6b2bd8560c) - https://answers.microsoft.com/en-us/windows/forum/windows_10-networking/windows-shows-no-internet-access-but-my-internet/2e9b593f-c31c-4448-b5d9-6e6b2bd8560c

**Comment by jedisct1** [comment]
If you're using `dnscrypt-proxy 2.0.45`, just uncomment `map_file` in the `[captive_portals]` section.

If you're using an older version, there's a lot to read here: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Windows-NCSI

Simple DNSCrypt is a independent project with its own repository: https://github.com/bitbeans/SimpleDnsCrypt

**Reply by claudio-klez** [reply]
Wow. Turns out I may not only have a super old version, but also may not have properly read the docs. Shameless me. Love dns crypt, BTW.
